Px.Utils/Validation/SyntaxValidation/SyntaxValidationUtilityMethods.cs

```diff
@@ -384,6 +399,137 @@ internal static int GetLineChangesValidity(string value, PxFileSyntaxConf syntax
             return -1;
         }
 
+        private static bool TryGetTimeValueFormat(string input, out ValueType? valueFormat, PxFileSyntaxConf syntaxConf)
+        {
+            // Value has to start with the time interval indicator (TLIST by default)
+            if (!input.StartsWith(syntaxConf.Tokens.Time.TimeIntervalIndicator))
+            {
+                valueFormat = null;
+                return false;
+            }
+
+
+            ExtractSectionResult intervalSection = ExtractSectionFromString(input, syntaxConf.Symbols.Value.TimeSeriesIntervalStart, syntaxConf.Symbols.Key.StringDelimeter, syntaxConf.Symbols.Value.TimeSeriesIntervalEnd);
+            // There can only be one time interval specifier section
+            if (intervalSection.Sections.Length != 1)
+            {
+                valueFormat = null;
+                return false;
+            }
+            // Depending on the format, interval section may have the time range specified. Interval token is always the first part of the section
+            string[] splitIntervalSection = intervalSection.Sections[0].Split(syntaxConf.Symbols.Value.ListSeparator);
+            string intervalToken = splitIntervalSection[0];
+            string? timeRange = splitIntervalSection.Length == 2 ? splitIntervalSection[1] : null;
+            if (!syntaxConf.Tokens.Time.TimeIntervalTokens.Contains(intervalToken))
+            {
+                valueFormat = null;
+                return false;
+            }
+            string remainder = intervalSection.Remainder.Remove(0, syntaxConf.Tokens.Time.TimeIntervalIndicator.Length);
+            return timeRange is not null ? TryGetTimeValueRangeFormat(timeRange, intervalToken, out valueFormat, syntaxConf) : TryGetTimeValueSeriesFormat(remainder, intervalToken, out valueFormat, syntaxConf);
+        }
+
+        private static bool TryGetTimeValueSeriesFormat(string input, string timeInterval, out ValueType? valueFormat, PxFileSyntaxConf syntaxConf)
```

Comment:
The return value of this function is never used, so the Try structure should be changed to a normal method.